packages/dds/tree/src/feature-libraries/README.md

```diff
@@ -1,6 +1,6 @@
 # Feature Libraries
 
-Concrete implementations of abstractions used to parameterize `SharedTreeCore`, or any other entry points into `tree`.
+Concrete implementations of abstractions used to parameterize `SharedTreeCore`, or any other entry points the `tree` package.
```

Comment:
```suggestion
Concrete implementations of abstractions used to parameterize `SharedTreeCore`, or any other entry points in the `tree` package.
```

Not sure if this is the right meaning.

Reply:
```suggestion
Concrete implementations of abstractions used to parameterize `SharedTreeCore`, or any other entry points to the `tree` package.
```

Reply:
I don't think this is gramatically correct.

Reply:
Seems fine to me, but maybe I missed something.

"any other entry points to the `tree` package" is a noun phrase and "Concrete implementations of abstractions used to parameterize `SharedTreeCore`" is a noun phrase, making this whole thing a noun phrase. While not a sentence, I think using a noun phrase as the documentation for a module to describe what it is reasonable.

"entry point to" is the common way to use "entry point". See the example usage in https://www.oxfordlearnersdictionaries.com/us/definition/english/entry-point

Reply:
Ah yeah, you're right. "entry points" was sounding like a noun + verb in my head.